Src/Service/Service/service/ProjectService.cs

```diff
@@ -40,24 +40,24 @@ public void AddProject(IProject project)
             UnitOfWork.Projects.Add(project);
         }
 
-        public IProjectApplication[] GetAllApplications(IProject project)
+        public IProjectApplication[] GetAllApplications(int projectId)

```

Comment:
I like having these as first class entities (like IProject) and just having the Presenter load the IProject first. Not a big deal though. If you want to keep it with a primitive in as the argument, I'd consider changing the name to projectRecordId.